### PR TITLE
Fix startup errors

### PR DESCRIPTION
**Pull request by justinmuskopf**

### CODE REVIEW

**Comment by sarahsturgeon** [comment]
@justinmuskopf Any reason not to merge this?

File: lua/autorun/server/sv_restrict_propcore.lua [hunk 1]
```diff
@@ -23,22 +23,26 @@ local function restrictPropCoreFunctions()
     }
 
     for _, signature in pairs( restrictedFunctions ) do
-        local oldFunc = wire_expression2_funcs[signature][3]
+        if wire_expression2_funcs then
```
Comment (sarahsturgeon) [inline lua/autorun/server/sv_restrict_propcore.lua:26]: CECO? (`continue` early `continue` often)

**Review by sarahsturgeon — CHANGES_REQUESTED** [review]

**Comment by sarahsturgeon** [comment]
@justinmuskopf ping

File: lua/autorun/server/sv_restrict_propcore.lua (hunk 1 above)
Comment (justinmuskopf) [reply lua/autorun/server/sv_restrict_propcore.lua:26]: sure, if you introduce the `continue` keyword into the language for me, boss

Comment (sarahsturgeon) [reply lua/autorun/server/sv_restrict_propcore.lua:26]: Well, do I have a surprise for you..

```lua
local a = {"this", "has", "many", "entries"}

for k, v in pairs(a) do
    print(v)
end
```
results in:
```
this
has
many
entries
```

however, this:

```lua
local a = {"this", "has", "many", "entries"}

for k, v in pairs(a) do
    if v ~= "this" then continue end
	
    print(v)
end
```
results in
```
this
```

**Comment by sarahsturgeon** [comment]
@justinmuskopf I implemented `continue` into glua for you please fix

**Pull request merged**